GenumKit/Stencil/Contexts.swift

```diff
@@ -131,3 +131,35 @@ extension StringsFileParser {
     return Context(dictionary: ["enumName": enumName, "tableName": tableName, "strings": strings])
   }
 }
+
+/* MARK: - Stencil Context for Fonts
+- `enumName`: `String`
+- `families`: `Array`
+  - `name`: `String`
+  - `fonts`: `Array`
+    - `name`: `String`
+    - `value`: `String`
```

Comment:
Feels strange that the `families.fonts.name` is not the font name but just the style, while `families.fonts.value` is not very explicit.

I suggest instead:
```
 - `enumName`: `String`
 - `families`: `Array`
   - `name`: `String` // Like "Arial" 
   - `variants`: `Array`
     - `style`: `String` // Like "Bold Italic"
     - `fontName`: `String` // Postscript name to use with UIFont(name:size:), like "Arial-BoldItalicMT"
```
Thoughts?